packages/components/src/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -56,14 +57,14 @@ export const CopyToClipboard: FC<CopyToClipboardProps> = (props) => {
   const { children = childrenText, content, success = successText } = props
 
   const [copied, setCopied] = useState(false)
-  const buttonRef = useRef<HTMLButtonElement>(null)
+  const [buttonRef, setButtonRef] = useState<HTMLButtonElement>()
```

Comment:
With `useForkedRef` in `MultiFunctionButton` this can go back to how it was before, which is a bit more performant:
```suggestion
  const buttonRef = useRef<HTMLButtonElement>(null)
```